openvdb_points/unittest/TestAttributeArray.cc

```diff
@@ -1675,6 +1679,86 @@ TestAttributeArray::testDelayedLoad()
     }
 }
 
+
+void
+TestAttributeArray::testQuaternions()
+{
+    using namespace openvdb;
+
+    typedef TypedAttributeArray<math::Quat<float> >         AttributeQF;
+    typedef TypedAttributeArray<QuatR>                      AttributeQD;
+
+    AttributeQF::registerType();
```

Comment:
Maybe, I'm trying not go too crazy with the type registrations, but I guess we should add them though.